README.md

```diff
@@ -7,6 +8,10 @@ To know more about Razorpay payment flow and steps involved, please read up here
 
 ## Installation
 
+This has 3 steps: add to project, installation and linking iOS SDK.
```

Comment:
I'd recommend the following headers
- Installation
- Linking
   - automatic linking 
   - manual linking
   - Linking IOS sdk